htdocs/js/jquery.contextualhover.js

```diff
@@ -285,6 +285,8 @@ _renderPopup: function() {
         if ( ! data.is_closed_membership || data.is_member ) {
             if ( data.is_member )
                 this._addAction( data.url_leavecomm, "Leave", "leave" );
+            else if ( data.is_invited )
+                this._addAction( data.url_acceptinvite, "Accept invitation", "accept");
```

Comment:
Every time I look at jQuery code, I'm impressed by how much simpler it is to read and write.  I really need to learn more about it. :)